src/model/generated/loan.model.ts

```diff
@@ -0,0 +1,36 @@
+import {Entity as Entity_, Column as Column_, PrimaryColumn as PrimaryColumn_, ManyToOne as ManyToOne_, Index as Index_} from "typeorm"
+import * as marshal from "./marshal"
+import {Currency, fromJsonCurrency} from "./_currency"
+import {Height} from "./height.model"
+
+@Entity_()
+export class Loan {
+    constructor(props?: Partial<Loan>) {
+        Object.assign(this, props)
+    }
+
+    @PrimaryColumn_()
+    id!: string
+
+    @Column_("jsonb", {transformer: {to: obj => obj.toJSON(), from: obj => obj == null ? undefined : fromJsonCurrency(obj)}, nullable: false})
+    token!: Currency
+
+    @Column_("text", {nullable: false})
+    userParachainAddress!: string
+
+    @Column_("numeric", {transformer: marshal.bigintTransformer, nullable: true})
+    amountBorrowed!: bigint | undefined | null
```

Comment:
For the next iteration, please add `amountBorrowedHuman` and `amountRepaidHuman` for the decimal representation of these values. But OK to merge like this!

We will be providing helper functions with this PR: https://github.com/interlay/interbtc-squid/pull/62